app/elements/challenge/challenge-detail.html

```diff
@@ -34,22 +34,22 @@
 	</style>
 	<template>
 		<iron-ajax
-			id="challengeDetailAjax"
-			loading="{{challengeLoading}}"
-			handle-as="json"
-			on-response="onResponse"
-			on-error="onError">
+				id="challengeDetailAjax"
+				loading="{{challengeLoading}}"
```

Comment:
Indented with two tabs where on would have been fine. It was good before this change.